plugins/inputs/nvidia_smi/schema_v12/parser.go

```diff
@@ -76,7 +76,9 @@ func Parse(acc telegraf.Accumulator, buf []byte) error {
 		common.SetIfUsed("int", fields, "clocks_current_memory", gpu.Clocks.MemClock)
 		common.SetIfUsed("int", fields, "clocks_current_video", gpu.Clocks.VideoClock)
 		common.SetIfUsed("float", fields, "power_draw", gpu.PowerReadings.PowerDraw)
+		common.SetIfUsed("float", fields, "power_limit", gpu.PowerReadings.PowerLimit)
 		common.SetIfUsed("float", fields, "power_draw", gpu.GpuPowerReadings.PowerDraw)
+		common.SetIfUsed("float", fields, "power_limit", gpu.GpuPowerReadings.PowerLimit)
```

Comment:
What is this one from?

Reply:
Well we defined this field already but did not output it. I guess some models will report the limit there?

Reply:
Ah totally missed that! thanks